server/server.js

```diff
@@ -88,6 +89,20 @@ app.get('/', (req, res) => {
   res.sendFile(renderIndex());
```

Comment:
This route is not being matched since it's already declared in server.routes. Even if it did get matched it would throw an error because you can't pass renderIndex() to sendFile().